PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,29 @@
-When opening a pull request to submit a new KIP, please use the suggested template: https://github.com/klaytn/KIPs/blob/master/kip-template.md
+## Proposed changes
 
-We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:
+- Describe your changes to communicate to the maintainers why we should accept this pull request.
+- If it fixes a bug or resolves a feature request, be sure to link to that issue.
 
- - The PR edits only existing draft PRs.
- - The build passes.
- - Your Github username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
- - If matching on email address, the email address is the one publicly listed on your GitHub profile.
+## Types of changes
+
+Please put an x in the boxes related to your change.
+
+- [ ] Bugfix
+- [ ] New feature or Enhancement
+- [ ] Others
```

Comment:
Could you remove or change the options? `Bugfix` option doesn't seem to be a good fit for KIP.